packages/manager/apps/pci-users/src/download-openrc.constants.ts

```diff
@@ -0,0 +1,19 @@
+export const OPENRC_VERSION: Record<string, string> = {
+  V2: 'v2.0',
+  V3: 'v3',
+};
+
+export const OPENSTACK_GUIDE: Record<string, Record<string, string>> = {
```

Comment:
same comment, do you really need to type it "Record<string, Record<string, string>>" ?

since you initialize the variable with a value, can't typescript automatically guess the correct type ?